internal/file/file.go

```diff
@@ -45,8 +45,14 @@ func HasExtension(path string, extensions []string) bool {
 // ShouldIgnore returns true if the path matches any of the ignore strings
 func ShouldIgnore(path string, ignoreFolders []string) bool {
 	segments := strings.Split(path, string(os.PathSeparator))
-	for _, segment := range segments {
-		for _, ignore := range ignoreFolders {
+	for _, ignore := range ignoreFolders {
+		if strings.Contains(ignore, string(os.PathSeparator)) {
+			if strings.Contains(path, ignore) {
```

Comment:
if **ignore** is: "ules/lab" then following paths will be ignored:
"modules/lab", "modules/laboratory/...", basically all paths matching: `/.*ules\/lab.*/`
![chrome_HvrFHvs6T2](https://user-images.githubusercontent.com/4062731/75376763-de0c2080-58d0-11ea-8867-d7a380492d43.png)

Reply:
That is sooo true ;)

Adding a failing test in 3..2..1..